db/migrate/20211103174459_create_invoices.rb

```diff
@@ -2,7 +2,7 @@ class CreateInvoices < ActiveRecord::Migration[5.2]
   def change
     create_table :invoices do |t|
       t.references :customer, foreign_key: true
-      t.string :status
+      t.integer :status
```

Comment:
I forget where we landed on this... would it be beneficial to set a default status as "in progress" for these enums? Or do we want to keep it as is? Might make life easier if we need to add more data down the road.

Reply:
good point! totally glossed over the default option, just added them in